metadata-jobs/mae-consumer-job/src/main/java/com/linkedin/metadata/kafka/config/HydratorFactoryConfig.java

```diff
@@ -22,6 +24,7 @@
   @Bean
   public HydratorFactory getHydratorFactory() {
     Client restClient = DefaultRestliClientFactory.getRestLiClient(gmsHost, gmsPort, gmsUseSSL, gmsSslProtocol);
-    return new HydratorFactory(restClient);
+    EntityClient entityClient = new EntityClient(restClient);
+    return new HydratorFactory(entityClient, SnapshotEntityRegistry.getInstance());
```

Comment:
This scares me - It leaks the concept of EntityRegistry to outside of GMS. I do not feel like it should be leaked in this way if we can avoid it

Reply:
It's used to get the keyaspect schema for the entity type. Is there a way to avoid this?